src/index.html

```diff
@@ -46,7 +48,7 @@ <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2
         class="card__button"
         href="#buy"
         data-qa="hover"
-      >buy</a>
+      >BUY</a>
```

Comment:
I think it looks better when the element text comes on the new line.
        <a
         href="#buy"
         class="product-card__link"
         data-qa="hover">
         BUY
        </a>

Reply:
not necessary, but yeah? doesn't hurt to do that:)